acados/utils/math.c

```diff
@@ -1057,7 +1057,7 @@ static void tql2(int_t dim, real_t *V, real_t *d, real_t *e)
     }
 }
 
-static void eigen_decomposition(int_t dim, real_t *A, real_t *V, real_t *d)
+void eigen_decomposition(int_t dim, real_t *A, real_t *V, real_t *d)
 {
     real_t *e = (real_t *) calloc(dim, sizeof(real_t));
```

Comment:
all dynamic memory allocation should be removed, if these functions are intended to be used by the core

Reply:
absolutely right! I will fix this now